caikit/runtime/client/utils.py

```diff
@@ -31,8 +33,45 @@ def construct_grpc_channel(
     target: str,
     options: Optional[List[Tuple[str, str]]] = None,
     tls: Optional[ConnectionTlsInfo] = None,
+    retries: Optional[int] = None,
```

Comment:
We don't have the args to this function in the `docstring`, but I think it might be good to add them since things like `options` and `retry_options` require specific formats that could use citations/docs.